src/Terminal/commands/runScript.ts

```diff
@@ -52,6 +52,12 @@ export function runScript(path: ScriptFilePath, commandArgs: (string | number |
   const success = startWorkerScript(runningScript, server);
   if (!success) return Terminal.error(`Failed to start script`);
 
+  if (path.endsWith(".script")) {
```

Comment:
We'll have to hunt these migration notices down to remove them once deprecation is final. 

Extract to a migration ts file or leave a comment so the search for them can be semi-automated.

Reply:
They are only in two places, the commonEditor function and the runScript function. Also it's extremely easy to find them by searching "migrate", I just checked.

Reply:
I agree with idea of extraction, for different reasons: we might want to adjust the message, or the criteria (maybe only the first time you edit?) and that should be kept in sync.

Reply:
I've moved it out into it's own file, that should make modifications trivial.